README.md

```diff
@@ -204,6 +204,20 @@ app.use(second, func)
 
 This is useful in cases where an injected variable from a plugin needs to be made available to another.
 
+If the meta data of a plugin contain a `onRegister(server, opts)` function, it will be executed synchronous before the plugin is being loaded: 
+
+```js
+async function plugin (app, opts) {
+  console.log(opts.list)
+})
+plugin[Symbol.for('plugin-meta')] = {
```

Comment:
A user could overwrite the `fastify-plugin` meta data when they define it afterwards. I would use a different symbol.

Reply:
While this is true, I think it’s the user responsibility to keep the previous info around when overriding.

I prefer not to add another symbol, as it might become harder to integrate later.